CODEOWNERS.md

```diff
@@ -1,2 +1,2 @@
 * @gyoshev @joneff
-docs/* @dmihaylo
+docs/ @dmihaylo
```

Comment:
As per https://help.github.com/articles/about-codeowners/, the modification will match all docs/ directories, as well as all their child directories.